docs: call out danger of caching live query responses (#15782)
sveltejs/kit: documentation/docs/20-core-concepts/60-remote-functions.md

As with `query` and `query.batch`, call `.run()` outside render when you need imperative access. For live queries, `run()` returns a `Promise<AsyncGenerator<T>>`.

> > [!NOTE] It's essential that you don't cache live query responses in a service worker, since by cloning them you will cause the stream to continue long after the page is closed. Make sure that your caching logic excludes any responses with a `Cache-Control` header that includes `no-store`.

Author: ```suggestion
> [!NOTE] It's essential that you don't cache live query responses in a service worker, since the cloned response will continue streaming long after the page is closed. Make sure that your caching logic excludes any responses with a `Cache-Control` header that includes `no-store`.
```
